### PR TITLE
Check metadata

### PR DESCRIPTION
**Pull request by nthfloor**

### CODE REVIEW

File: metadata_finds/metadata_finds.sh [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+read continuar
+
+if [ $continuar == "n" ]
+        then
+                exit 1
+fi
+declare -A queries
+
+####  the bellow  queries  will used to run in mentioned partners databases  ####
+#--Get List of all programs
```
Comment (nthfloor) [inline metadata_finds/metadata_finds.sh:117]: Are we missing concepts here?

File: metadata_finds/metadata_finds.sh [hunk 2]
```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
```
Comment (nthfloor) [inline metadata_finds/metadata_finds.sh:1]: @psmatsinhe this script will need some documentation or a user guide to explain on all the steps for the partners.

File: metadata_finds/metadata_finds.sh [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+queries[1,0]="SELECT p.program_workflow_id,p.program_id,p.retired program_workflow_retired, p.concept_id,c.name,p.uuid program_workflow_uuid,c.uuid concept_uuid FROM program_workflow p INNER JOIN concept_name c ON p.concept_id=c.concept_id ORDER BY program_workflow_id"
+queries[1,1]="all-workflow"
+
+#--Get all the workflow states
+queries[2,0]="SELECT p.program_workflow_id,p.program_Workflow_state_id,p.retired program_workflow_state_retired, p.concept_id, c.name, p.uuid program_workflow_state_uuid FROM program_workflow_state p INNER JOIN concept_name c ON p.concept_id=c.concept_id ORDER BY program_workflow_id"
+queries[2,1]="all-workflow-states"
+
+#--Get all the person attribute types
+queries[3,0]="SELECT person_attribute_type_id, name,retired,uuid FROM  person_attribute_type ORDER BY person_attribute_type_id"
+queries[3,1]="all-person-attributes"
```
Comment (nthfloor) [inline metadata_finds/metadata_finds.sh:131]: Do you think we should maybe also include relationship types?

File: metadata_finds/metadata_finds.sh (hunk 2 above)
Comment (psmatsinhe) [reply metadata_finds/metadata_finds.sh:1]: Yes, we need to specify how to run, some kind of SOP

File: metadata_finds/metadata_finds.sh (hunk 1 above)
Comment (psmatsinhe) [reply metadata_finds/metadata_finds.sh:117]: I was not thinking on getting them with a believe that all partners except ICAP have the same concept dictionary as been proven with latest forms.... You think we need to get them?

File: metadata_finds/metadata_finds.sh (hunk 3 above)
Comment (psmatsinhe) [reply metadata_finds/metadata_finds.sh:131]: Did not get that, what kind of relationship?

File: metadata_finds/metadata_finds.sh (hunk 1 above)
Comment (nthfloor) [reply metadata_finds/metadata_finds.sh:117]: Sure but shouldnt we still check that this is in fact the case?

File: metadata_finds/metadata_finds.sh (hunk 3 above)
Comment (nthfloor) [reply metadata_finds/metadata_finds.sh:131]: Well depends I guess, but on the metadata server we have the following:
Screenshot from 2019-07-09 10-57-40

File: metadata_finds/metadata_finds.sh (hunk 2 above)
Comment (nthfloor) [reply metadata_finds/metadata_finds.sh:1]: SOP can be found here: https://docs.google.com/document/d/1Kj7IbXzVyWimnDtNYEzFWlJaPFj6nbEGASnYuCWM-dc/edit

File: metadata_finds/metadata_finds.sh (hunk 1 above)
Comment (nthfloor) [reply metadata_finds/metadata_finds.sh:117]: I've added the concepts to the script, jsut in case we need them :)

**Pull request merged**